backend/de.metas.contracts/src/main/java/de/metas/contracts/interceptor/C_Order.java

```diff
@@ -136,10 +136,22 @@ private void extendFlatrateTermIfAutoExtension(final I_C_Flatrate_Term term)
 		}
 	}
 
-	@DocValidate(timings = { ModelValidator.TIMING_AFTER_COMPLETE, ModelValidator.TIMING_AFTER_REACTIVATE })
-	public void handleReactivate(final I_C_Order order)
+	@DocValidate(timings = ModelValidator.TIMING_AFTER_REACTIVATE )
+	public void handleReactivate(@NonNull final I_C_Order order)
 	{
-		final List<I_C_OrderLine> orderLines = orderDAO.retrieveOrderLines(order, I_C_OrderLine.class);
+		final DocTypeId docTypeId = orderBL.getDocTypeIdEffectiveOrNull(order);
+		if (docTypeId == null)
+		{
+			return;
+		}
+
+		final String docSubType = docTypeBL.getById(docTypeId).getDocSubType();
```

Comment:
better add a method like `isStandardOrder(DocTypeId)` to `de.metas.document.IDocTypeBL`

Reply:
added